lib/lotus/view/rendering/partial_templates_finder.rb

```diff
@@ -0,0 +1,49 @@
+require 'lotus/view/template'
+
+module Lotus
+  module View
+    module Rendering
+      # Find partial templates in the file system
+      #
+      # @api private
+      # @since x.x.x
+      #
+      # @see View::Template
+      class PartialTemplatesFinder
+        # Search pattern for partial file names
+        #
+        # @api private
+        # @since x.x.x
+        PARTIAL_PATTERN    = '_*'.freeze
+
+        # Initialize a configuration instance
+        #
+        # @return [Array] array of arrays containing partial template name,
+        # format and a newly created View::Template
+        #
+        # @since x.x.x
+        def self.find_partials(path)
+          _find_partials(path).map do |template|
+            path_name = Pathname(template)
+            partial_path, partial_base_name = Pathname(template).relative_path_from(path).split
+            partial_base_parts = partial_base_name.to_s.split('.')
+            ["#{partial_path}#{::File::SEPARATOR}#{partial_base_parts[0]}", partial_base_parts[1], View::Template.new(template)]
```

Comment:
@stevehook You should probably let `Configuration` to invoke this class like the following snippet, so you can pass easily access the `Configuration#default_encoding` information.

```ruby
Lotus::View::Rendering::PartialTemplatesFinder.new(self, root).find
```

In the code above `self` is an instance of `Configuration`. This refers to the new implemented method `Configuration#load_partials!`.